test/Microsoft.Azure.WebJobs.Host.EndToEndTests/BlobBindingEndToEndTests.cs

```diff
@@ -94,7 +94,7 @@ public BlobBindingEndToEndTests(TestFixture fixture)
             var ex = await Assert.ThrowsAsync<FunctionInvocationException>(() =>
            _fixture.Host.CallAsync(typeof(BlobBindingEndToEndTests).GetMethod("CloudBlockBlobBinding_WithUrlBinding"), arguments));
             // CloudBlockBlobBinding_WithUrlBinding is suppose to bind to a blob
-            Assert.Equal($"Invalid absolute blob url: {poco.A}", ex.InnerException.InnerException.Message);
```

Comment:
Can we keep the bit verifying that the error message is as expected?

Reply:
We don't know the user is trying to give us a URL, so it's ambiguus what error message we specifically get. 

---
In reply to: [151538959](https://github.com/Azure/azure-webjobs-sdk/pull/1422#discussion_r151538959) [](ancestors = 151538959)